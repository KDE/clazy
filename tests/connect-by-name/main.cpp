#include <QtCore/QObject>

class MyObj; // OK

class MyObj : public QObject
{
public Q_SLOTS:
    void on_foo_bar();  // Warn

public:
    void on_foo2_bar2(); // OK

signals:
    void on_foo3_bar3(); // OK
};

class MyObj; // OK

void MyObj::on_foo_bar() // OK
{
}

void MyObj::on_foo2_bar2() // OK
{
}
