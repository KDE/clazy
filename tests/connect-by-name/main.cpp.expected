connect-by-name/main.cpp:8:5: warning: Slots named on_foo_bar are error prone [-Wclazy-connect-by-name]
